test/node/BlinkIDLFileDAO-integration.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('BlinkIDLFileDAO integration', function() {
+  var execSync = require('child_process').execSync;
+
+  var IDLFileContents;
+  var StoreAndForwardDAO;
+  var BlinkIDLFileDAO;
+  var localRepositoryPath;
+  var includePaths;
+
+  beforeAll(function() {
+    localRepositoryPath =
+        require('path').resolve(__dirname, '../data/blink/git');
+
+    var opts = {cwd: localRepositoryPath};
+    execSync('/usr/bin/git init', opts);
```

Comment:
You don't clean this up, so is it possible that it could have side effects on other tests?